plugins/inputs/fibaro/fibaro.go

```diff
@@ -69,11 +69,12 @@ type Devices struct {
 	Type       string `json:"type"`
 	Enabled    bool   `json:"enabled"`
 	Properties struct {
-		Dead   interface{} `json:"dead"`
-		Energy interface{} `json:"energy"`
-		Power  interface{} `json:"power"`
-		Value  interface{} `json:"value"`
-		Value2 interface{} `json:"value2"`
+		BatteryLevel interface{} `json:"batteryLevel"`
```

Comment:
I realize you're just continuing the pattern here, but it's unclear to me why these are interface{} and not floats, bools, etc. Have any insight?